build/templates/_converters.py.mako

```diff
@@ -8,14 +8,70 @@ import datetime
 import six
 
 
+def _repeated_capability_string_to_list(channels, prefix):
```

Comment:
function says "repeated capability string" but parameter name is channels. In our heads it's more or less the same thing, but we should use consistent terminology. Please rename parameter.

Reply:
Changed terminology